mungegithub/mungers/flake-manager.go

```diff
@@ -96,7 +111,9 @@ func (p *FlakeManager) EachLoop() error {
 }
 
 // AddFlags will add any request flags to the cobra `cmd`
-func (p *FlakeManager) AddFlags(cmd *cobra.Command, config *github.Config) {}
+func (p *FlakeManager) AddFlags(cmd *cobra.Command, config *github.Config) {
+	cmd.Flags().StringVar(&p.ownerPath, "test-owners-config", "", "file containing a CSV-exported test-owners spreadsheet")
```

Comment:
This is OK as a first step but we really need to get this from the repository. We can't restart the submit queue every time this changes.

@eparis might have ideas